app/src/main/java/com/duckduckgo/app/global/view/ClearPersonalDataAction.kt

```diff
@@ -81,8 +85,10 @@ class ClearPersonalDataAction(
         shouldFireDataClearPixel: Boolean
     ) {
         withContext(Dispatchers.IO) {
+            val fireproofWebsites = fireproofWebsiteRepository.fireproofWebsitesSync().map { it.domain }
             cookieManager.flush()
             geoLocationPermissions.clearAllButFireproofed()
+            sitePermissionsManager.clearAllButFireproof(fireproofWebsites)
```

Comment:
Good question. In this case, I needed fireproof websites passed as an argument because it's a different module and I don't have access to Fireproof Repository.